Component/DatePicker.js

```diff
@@ -0,0 +1,64 @@
+//Date Picker
+
+import React, { useState } from "react";
+import { Button, Platform, StyleSheet, Switch, Text, View } from "react-native";
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+
+const MainPage = () => {
+  const [pickerMode, setPickerMode] = useState(null);
+  const [inline, setInline] = useState(false);
+
+  const showDatePicker = () => {
+    setPickerMode("date");
+  };
+  
+  const hidePicker = () => {
+    setPickerMode(null);
+  };
```

Comment:
You can make this one function. The function will check the pickerMode then set it appropriately. 

e.g:

`const handleDatePicker = () => pickerMode === null ? setPickerMode('date') : setPickerMode(null)`

This will make your onConfirm and onCancel be the same function

Reply:
ok yes let me try with creating a function